debugging/Makefile

```diff
@@ -1,5 +1,5 @@
 # Golang EOL overview: https://endoflife.date/go
-DOCKER_GOLANG_IMAGE ?= golang:1.19
+DOCKER_GOLANG_IMAGE ?= golang:1.21.0-bullseye
```

Comment:
Should use the same go version as building, `1.20`.

Reply:
doesn't seem to matter, did you try with 1.20?